lib/schemas/privateThemeConfig.json

```diff
@@ -1,5 +1,4 @@
 {
-  "$schema": "http://json-schema.org/draft-04/schema#",
   "id": "http://themes.bigcommerce.com/theme_packages/config",
```

Comment:
Is this schema different from `lib/schemas/themeConfig.json`? If so, it should have a different `id`.